src/dvc_objects/fs/local.py

```diff
@@ -113,7 +113,10 @@ def rm_file(self, path):
         remove(path)
 
     def rm(self, path, recursive=False, maxdepth=None):
-        remove(path)
+        if not isinstance(path, list):
+            path = [path]
```

Comment:
can we change this around so that it's
```
if isinstance(path, str):
    path = [path]
```

I know the fsspec type is `List` but it makes more sense to be allowed to pass in other iterable types as well.